project-koku/koku
# effective cost fixing memory usage sql ordering

Fixes [COST-2217]
Changes proposed in this PR:
* Fixing tiny ordering mistake

Testing Instructions:
1. add ocp source
2. check memory requests are correct.
